### PR TITLE
Feature/ab#8819 applicant mapping

### PR DESCRIPTION
**Pull request by jimmyPasta**

### CODE REVIEW

File: applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Unity.GrantManager.Domain.csproj [hunk 1]
```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Unity.GrantManager.Application.Contracts\Unity.GrantManager.Application.Contracts.csproj" />
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Unity.GrantManager.Domain.csproj:12]: The domain project shouldn't reference the application contracts project - looking at how to refactor this

Comment (AndreGAot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Unity.GrantManager.Domain.csproj:12]: @jimmyPasta - please see branch:
https://github.com/bcgov/Unity/tree/feature/AB%238819-Applicant-Mapping-Refactor
I tried to rework the code to keep the domain layer clean and not reference the contracts project, will need some testing to make sure the logic is still sound

Comment (jimmyPasta) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Unity.GrantManager.Domain.csproj:12]: Ok Thanks Should I merge 238819 into this one?

Comment (AndreGAot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Unity.GrantManager.Domain.csproj:12]: yeah, if you are happy with the changes and i didn't break anything

**Review by AndreGAot — APPROVED** [review]
Ready for Dev

**Pull request merged**